kubetest2-gke/deployer/up.go

```diff
@@ -38,6 +38,16 @@ func (d *deployer) Up() error {
 		return err
 	}
 
+	defer func() {
+		if d.RepoRoot == "" {
+			klog.Warningf("repo-root not supplied, skip dumping cluster logs")
```

Comment:
I feel this should be `Info` since it's normal for non-k8s projects to skip dumping clusters logs.

Reply:
Usually info comments are used for more verbose explanation of steps being carried out.
Whereas here we'll be missing a key part of the deployer lifecycle, feels like it should be a warning.